addons/binding/org.openhab.binding.dwdunwetter/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="dwdunwetter"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Sample Thing Type -->
+	<thing-type id="dwdwarnings">
+		<label>DwdUnwetter Binding Thing</label>
+		<description>Binding für den Abruf von DWD Unwetterwarnungen</description>
+		<config-description>
+			<parameter name="cellId" type="text" required="true">
+				<label>Cell-ID</label>
+				<description><![CDATA[ID der abzufragenden Zelle.
+Siehe https://www.dwd.de/DE/leistungen/opendata/help/warnungen/cap_warncellids_csv.csv.
+Es kann auch mittels % eine Gesamtmenge abgefragt werden, z.B. 8111% alle Gemeinden die mit 8111 anfangen.]]></description>
+			</parameter>
+			<parameter name="refresh" type="integer" required="true" min="15">
+				<default>30</default>
+				<label>Refresh in minutes</label>
+				<description>Abfrageintervall in Minuten. Minimal 15 Minuten.</description>
```

Comment:
Please use English only in this thing-types.xml file so non-German speakers can also review the modeling of your binding. 

See for instance the Tankerkönig Binding on how to add translations:
* It uses English in the thing type definitions [station.xml](https://github.com/openhab/openhab2-addons/blob/master/addons/binding/org.openhab.binding.tankerkoenig/ESH-INF/thing/station.xml), [webservice.xml](https://github.com/openhab/openhab2-addons/blob/master/addons/binding/org.openhab.binding.tankerkoenig/ESH-INF/thing/webservice.xml).
* The German translations are added in the [tankerkoenig_de.properties](https://github.com/openhab/openhab2-addons/blob/master/addons/binding/org.openhab.binding.tankerkoenig/ESH-INF/i18n/tankerkoenig_de.properties) file.

For more on this see also the Eclipse SmartHome [Internationalization documentation](https://www.eclipse.org/smarthome/documentation/features/internationalization.html).

Reply:
You are right, I might have looked at an old version or my memories are wrong. Will translate everything to english (and while doing so realized that my mix of german and english in the xml was a really bad idea)